src/typeDefs/scalars.ts

```diff
@@ -11,9 +12,35 @@ export const scalars = gql`
   scalar ID
   scalar PhoneNumber
   scalar PositiveInt
+  scalar StartDate
   scalar Time
   scalar URL
   scalar Upload
   scalar JSON
   scalar Any
 `;
+export const StartDateResolver = new GraphQLScalarType({
```

Comment:
Remove the new scalar and do comparisions on `startDate` and `endDate` within the resolver. Scalars are meant for extending the default set of scalars provided by graphql, not for putting business logic in the schema.